bowtie/_core.py

```diff
@@ -538,6 +543,47 @@ def supports(self, *dialects: Dialect) -> bool:
         """
         return self.info.dialects.issuperset(dialects)
 
+    async def get_versions(self):
+        from github3 import GitHub  # type: ignore[reportMissingTypeStubs]
+        from github3.exceptions import (  # type: ignore[reportMissingTypeStubs]
+            GitHubError,
+        )
+        from github3.models import (  # type: ignore[reportMissingTypeStubs]
+            GitHubCore,
+        )
+        from sortedcontainers import (  # type: ignore[reportMissingTypeStubs]
```

Comment:
You're adding a new dependency here, but haven't add it to the requirements in `pyproject.toml`.

Reply:
I'm not sure it's even super important for this to be sorted by this method, you probably could just not use this library and return a normal set, and sort it when/if you want to print it.